### PR TITLE
Added Glow Berries Being Useful

### PR DESCRIPTION
**Pull request by TheWireLord**
## Description

Adding usability to the Glow Berries which were added in the 1.17 update.
## Proposed changes

+ Added Glow Berry Juice as a result of using Glow Berries in the Juicer. (With the same stats as Sweet Berry Juice)
+ Added Glow Berries as a fuel in the Bio Reactor. (With the same stats as Sweet Berries).
+ Updated Wiki to include Glow Berry Juice.
## Checklist

- [x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [x] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [x] I followed the existing code standards and didn't mess up the formatting.
- [x] I did my best to add documentation to any public classes or methods I added.
- [x] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [x] I added sufficient Unit Tests to cover my code.

### CODE REVIEW

**Review by TheBusyBiscuit — APPROVED** [review]
Pretty much looking good to me, thank you!

Although a part of me wished glow berry juice gave night vision XD

**Comment by Sefiraat** [comment]
> Pretty much looking good to me, thank you!
> 
> Although a part of me wished glow berry juice gave night vision XD

I was going to say it should, surely, give the glowing effect :D?

**Review by Sefiraat — APPROVED** [review]
Looking fine, though swapping out for Night Vision or Glowing would make far more sense given the item, imo

**Comment by TheWireLord** [comment]
> Pretty much looking good to me, thank you!
> 
> Although a part of me wished glow berry juice gave night vision XD

I had also thought of this, but it looked like the suggestion was not super popular in the Suggestions channel in Discord.
I feel that the Night Vision may be too much and may replace the potion, but I could see the Glowing effect be a possibility. I'm going to look into that.

**Comment by Sefiraat** [comment]
Personally I am in favour of what @Sfiguz7 mentioned on Discord, changing/overloading the SlimefunItemStack constructor to allow for multiple PotionEffects. Then perhaps all give a basic level of saturation and then specific ones could give additional effects based on their theme. However to avoid scope slipping, perhaps just make this one `GLOWING` for the time being and then it could be expanded on in a future PR?

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/generators/BioGenerator.java [hunk 1]
```diff
@@ -54,6 +54,7 @@ protected void registerDefaultFuelTypes() {
         registerFuel(new MachineFuel(2, new ItemStack(Material.SEA_PICKLE)));
         registerFuel(new MachineFuel(1, new ItemStack(Material.BAMBOO)));
         registerFuel(new MachineFuel(2, new ItemStack(Material.SWEET_BERRIES)));
+        registerFuel(new MachineFuel(2, new ItemStack(Material.GLOW_BERRIES)));
```
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/generators/BioGenerator.java:57]: These are already added below (being the version check)

```suggestion
```

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

**Review by TheBusyBiscuit — APPROVED** [review]

**Pull request merged**